Infinum/Tests/Shortcodes/DisallowDoShortcodeUnitTest.inc

```diff
@@ -5,5 +5,6 @@ echo do_shortcode( '' ); // Warning.
 echo do_shortcode( '[iscorrect]' . $text_to_be_wrapped_in_shortcode . '[/iscorrect]' ); // Warning.
 add_filter( 'widget_text', 'do_shortcode' ); // Warning.
 echo do_shortcode( '[contact-form-7 id="91" title="quote"]' ); // Warning.
+echo do_shortcode('[contact-form-7 id="91" title="quote"]' ); // Warning.
```

Comment:
```suggestion
echo do_shortcode( '[contact-form-7 id="91" title="quote"]' ); // Warning.
```

Reply:
This is a unit test, I didn't apply styling fixes on this (otherwise this whole file would be checked, and all the files shouldn't have extra spaces according to new rules) 🙂 .